lms/djangoapps/certificates/tests/test_cert_management.py

```diff
@@ -185,14 +185,18 @@ def test_clear_badge(self, issue_badges, xqueue):
         args = u'-u {} -c {}'.format(self.user.email, text_type(key))
         call_command(self.command, *args.split(' '))
 
-        xqueue.return_value.regen_cert.assert_called_with(
+        self.assertEqual(xqueue.return_value.regen_cert.call_args.args, (
             self.user,
-            key,
-            course=self.course,
-            forced_grade=None,
-            template_file=None,
-            generate_pdf=True
-        )
+            key
+        ))
+        regen_cert_call_kwargs = xqueue.return_value.regen_cert.call_args.kwargs
+        self.assertEqual(regen_cert_call_kwargs.pop('course').location, self.course.location)
```

Comment:
Need to compare the location instead of the object (which is going to be different because it is loaded again).